package.json

```diff
@@ -20,97 +20,90 @@
     "cli-color": "1.2.0",
     "cli-spinner": "0.2.7",
     "commander": "2.12.2",
     "connect-livereload": "0.6.0",
     "connect-modrewrite": "^0.10.2",
     "deep-diff": "0.3.8",
-    "emphasize": "2.0.0",
+    "emphasize": "7.0.0",
     "enzyme": "3.3.0",
-    "enzyme-adapter-react-16": "1.8.0",
-    "eslint-config-airbnb-base": "13.1.0",
+    "enzyme-adapter-react-16": "1.15.8",
+    "eslint-config-airbnb-base": "15.0.0",
     "eslint-config-binary": "1.0.2",
     "eslint-config-prettier": "2.9.0",
     "eslint-import-resolver-alias": "1.1.1",
-    "eslint-plugin-import": "2.14.0",
-    "eslint-plugin-react": "7.11.1",
+    "eslint-plugin-import": "2.25.3",
+    "eslint-plugin-react": "7.28.0",
     "estraverse": "4.2.0",
     "extend": "3.0.2",
     "gettext-parser": "1.3.0",
     "grunt": "^1.6.1",
-    "grunt-babel": "7.0.0",
+    "grunt-babel": "8.0.0",
     "grunt-check-dependencies": "1.0.0",
-    "grunt-concurrent": "2.3.1",
-    "grunt-contrib-clean": "1.1.0",
-    "grunt-contrib-connect": "1.0.2",
+    "grunt-concurrent": "3.0.0",
+    "grunt-contrib-clean": "2.0.1",
+    "grunt-contrib-connect": "4.0.0",
     "grunt-contrib-copy": "1.0.0",
-    "grunt-contrib-cssmin": "3.0.0",
+    "grunt-contrib-cssmin": "5.0.0",
     "grunt-contrib-watch": "1.1.0",
-    "grunt-eslint": "21.0.0",
-    "grunt-gh-pages": "3.1.0",
+    "grunt-eslint": "24.3.0",
+    "grunt-gh-pages": "4.0.0",
     "grunt-mocha-test": "0.13.3",
     "grunt-notify": "0.4.5",
     "grunt-postcss": "^0.9.0",
     "grunt-sass": "3.0.2",
     "grunt-shell": "2.1.0",
-    "grunt-stylelint": "0.18.0",
-    "grunt-webpack": "3.1.3",
-    "jsdom": "^9.12.0",
+    "grunt-stylelint": "0.19.0",
+    "grunt-webpack": "6.0.0",
+    "jsdom": "9.12.0",
     "jsdom-global": "2.1.1",
-    "load-grunt-config": "^3.0.1",
+    "load-grunt-config": "^4.0.1",
     "load-grunt-tasks": "3.5.2",
-    "mocha": "5.2.0",
+    "mocha": "10.3.0",
     "mock-local-storage": "1.1.7",
-    "node-gettext": "2.0.0",
-    "node-sass": "4.12.0",
-    "postcss-scss": "0.4.1",
+    "node-gettext": "3.0.0",
+    "postcss-scss": "4.0.9",
     "react-render-html": "0.6.0",
-    "react-svg-loader": "3.0.1",
+    "sass": "^1.62.1",
     "sitemap": "1.13.0",
-    "stylelint": "10.0.0",
+    "stylelint": "15.11.0",
     "stylelint-formatter-pretty": "1.0.3",
-    "stylelint-selector-bem-pattern": "2.1.1",
-    "svgo": "1.2.1",
-    "sw-precache-webpack-plugin": "0.11.5",
+    "stylelint-selector-bem-pattern": "3.0.1",
+    "svgo": "3.2.0",
+    "terser-webpack-plugin": "5.1.1",
     "time-grunt": "1.4.0",
-    "uglifyjs-webpack-plugin": "1.3.0",
-    "webpack": "4.42.1",
-    "webpack-bundle-analyzer": "3.6.1",
-    "webpack-dev-server": "3.1.14",
-    "webpack-manifest-plugin": "2.0.4",
+    "webpack": "5.81.0",
+    "webpack-bundle-analyzer": "4.3.0",
     "webpack-merge": "4.1.4",
-    "ws": "7.0.0"
+    "ws": "8.16.0"
   },
   "dependencies": {
+    "@babel/polyfill": "7.12.1",
```

Comment:
this needs to move to dev deps

Reply:
According to the docs, it should be installed in the regular 'dependencies' (no --save-dev flag):
<img width="1349" alt="Screenshot 2024-03-21 at 11 59 58 AM" src="https://github.com/deriv-com/smarttrader/assets/103177211/0d15af6c-0ea9-4d4f-aafa-5447c77d3488">
I replaced babel-polyfill with it. babel-polyfill was also inside the regular 'dependencies'.
Should I still move it to dev dependencies?